boards/feather-m0/Makefile.include

```diff
@@ -0,0 +1,25 @@
+# define the cpu used by Adafruit Feather M0 boards
+export CPU = samd21
+export CPU_MODEL = samd21g18a
+
+#export needed for flash rule
+export PORT_LINUX ?= /dev/ttyUSB0
```

Comment:
@aabadie Mine is `/dev/ttyACM0`

Reply:
@aabadie Maybe this two patches would fix this?

[makefiles/tools/bossa: Introduce PORT_BOSSA variable](https://github.com/ynezz/RIOT/commit/6e7eb92ce331d5864a8cb7619b50b38446787a72)
[boards/feather-m0: Use PORT_BOSSA variable for flashing using Bossa](https://github.com/ynezz/RIOT/commit/52f9b6fc947665718211f0e96dc1a832722761e1)

Reply:
```
make -j10 BOARD=feather-m0 -C examples/hello-world flash term
...
/opt/devel/mcu/RIOT/dist/tools/bossa/bossac -p /dev/ttyACM0 -e -i -w -v -b -R /opt/devel/mcu/RIOT/examples/hello-world/bin/feather-m0/hello-world.bin
...
Done in 0.059 seconds
CPU reset.
...
/opt/devel/mcu/RIOT/dist/tools/pyterm/pyterm -p "/dev/ttyUSB0" -b "115200"
2017-09-28 15:21:58,683 - INFO # Connect to serial port /dev/ttyUSB0
Welcome to pyterm!
Type '/exit' to exit.
2017-09-28 15:21:59,688 - INFO # main(): This is RIOT! (Version: 2017.10-devel-509-g52f9b-ntbk-feather-m0-testing)
```

Reply:
Yes, I was thinking of something similar. Thanks, I'll update the PR asap